### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by quelchx**
file path error (spelling)

### CODE REVIEW

**Comment by quelchx** [comment]
hope this helps =)

File: pages/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default function HomePage() {
       WebViewer(
         {
           path: '/webviewer/lib',
-          initialDoc: '/files/pdftron_about.pdf',
+          initialDoc: "/files/PDFTRON_about.pdf",
```
Comment (mparizeau-pdftron) [inline pages/index.js:11]: Thanks! Can you just switch this to single quotes to be consistent with the rest of the code?
```suggestion
          initialDoc: '/files/PDFTRON_about.pdf',
```

**Review by mparizeau-pdftron — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 1 above)
Comment (quelchx) [reply pages/index.js:11]: Yeah I can do that no problem.

**Comment by quelchx** [comment]
changed it to single quotes -- should be good to go now.

**Review by mparizeau-pdftron — APPROVED** [review]

**Pull request merged**